### ISSUE TITLE
I don't understand how to exclude identifiers in the VendorPrefixes rule

### ISSUE THREAD

**Issue by himedlooff** [causes/scss-lint#379]
Here's my config:

```yml
  VendorPrefixes:
    enabled: true
    severity: warning
    identifier_list: base
    include: []
    exclude:
      - keyframes
```

Here's the SCSS:

```scss
@-webkit-keyframes anim {
    0% { opacity: 0; }
}
```

But I'm still getting the following linting error: "Avoid vendor prefixes."

**Comment by sds** [comment]
Thanks for the report, @himedlooff.

This is an unfortunate oversight. This linter uses reserved config parameters `include` and `exclude`, which are intended to be used to specify which files the linter should/should not run on.

In this case, the `keyframes` entry is being interpreted as a file name, not an identifier. 

I've fixed this issue in 7edbc38 by renaming the options to `additional_identifiers` and `excluded_identifiers`, respectively. Since this is a breaking change, I've also gone ahead and renamed the linter to `VendorPrefix` so that it matches the convention used by our other linters (i.e. singular instead of plural).

**Comment by himedlooff** [comment]
:smile: thanks @sds!
